articles/faq/can-i-switch-to-old-plan

```diff
@@ -0,0 +1,3 @@
+**Can I buy or switch account to one of the old plans that don’t include support?**
```

Comment:
Per discussion, add metadata. See the [example file](https://raw.githubusercontent.com/SparkPost/support-docs/master/example/article.md) for reference.